data/avgi/avgi 1 first contact.txt

```diff
@@ -0,0 +1,1055 @@
+			choice
+				`	(Attempt to land regardless)`
+					goto explode
+					to display
+						"flagship attribute: shields" + "flagship attribute: hull" <= 9000 * 1000
+				`	(Attempt to land regardless)`
+					goto hit
+					to display
+						"flagship attribute: shields" + "flagship attribute: hull" > 9000 * 1000
+						"flagship attribute: shields" + "flagship attribute: hull" < 60000 * 1000
+				`	(Attempt to land regardless)`
+					goto scratched
+					to display
+						"flagship attribute: shields" + "flagship attribute: hull" >= 60000 * 1000
+				`	(Move to a higher orbit)`
+					goto orbit
+
+			label explode
+			`	You make it no more than a few seconds before your ship is lanced by a rapid succession of shots from below. Your shields shatter almost instantly, your hull pierced as though it is tissue paper. You don't even have time to think before you are consumed by the rapidly expanding cloud of plasma that was once the <ship>.`
+				explode
+
+			label hit
+			`	You make it no more than a few seconds before the <ship> is lanced by a rapid succession of shots from below. Your ship appears to be holding up under the fire, but the species that lives here doesn't seem willing to welcome you. When you detect the launch of a rather unreasonable number of missiles, you decide it would be safer to proceed back to orbit.`
+				goto orbit
+
+			label scratched
+			`	You make it no more than a few seconds before the <ship> is lanced by a rapid succession of shots from below. Your ship easily shrugs off the kinetic impacts, but the species that lives here doesn't seem willing to welcome you. When you detect the launch of a rather unreasonable number of missiles, you decide it would be safer to proceed back to orbit.`
+				goto orbit
+
+			label uninhabited
+			`While this world looks presently unsettled, you can clearly make out signs that it has been explored, and even exploited, by an alien civilization. However, the signs also show that something disastrous happened here. Ruined satellites litter low orbit, and you can detect the broken remnants of surface outposts smeared across the land below. You take the <ship> in for a close pass over one of the structures, and can see small craters and patches of glassed terrain scattered around it.`
+				goto orbit
+
+			label orbit
+			`	Back in orbit, you are startled when another ship appears over the horizon on an intercept trajectory with you. It doesn't look like any other ship you've seen before, appearing more than a little ungainly compared to the sleeker human ships you're accustomed to. You aren't sure whether it belongs to the same species that inhabits this space, or an entirely different one, but it at least definitely isn't similar to the hostile ships that seem to infest this star cluster.`
+			choice
+				`	(Try transmitting a standard hail.)`
+					goto talk
+				`	(Transmit a series of prime numbers.)`
+					goto prime
+				`	(Remain silent.)`
+					goto silent
+
+			label station
```

Comment:
Is the following text supposed to be used for all stations regardless of if they are inhabited?